.github/workflows/e2e_android.yml

```diff
@@ -9,6 +9,9 @@ on:
     branches:
       - 'dependabot/**'
   workflow_dispatch:
+  workflow_run:
+    workflows: ['[TestRail] Start RC Run']
```

Comment:
If this needs to run whenever a release build is made, shouldn't this be `[Build] Release Build`?

Reply:
I've updated the scripts based on your additional comments.